languages/wordpress-indieweb.pot

```diff
@@ -268,153 +268,88 @@ msgid ""
-"service. Bridgy allows you to pull responses back from other sites as well "
-"as publish them"
+#: includes/getting-started.php:20
+msgid "Plugins"
 msgstr ""
 
-#: includes/getting-started.php:38
+#: includes/getting-started.php:22
 msgid ""
-"<strong>Syndication Links</strong> - Adds fields to a post to allow manual "
-"entry of syndication links as well as automatically from a supported "
-"syndication plugin. Fully supports <a "
-"href=\"https://github.com/crowdfavorite/wp-social\" "
-"target=\"_blank\">Social</a>, partial support for <a "
-"href=\"https://wordpress.org/plugins/social-networks-auto-poster-facebook-"
-"twitter-g/\" target=\"_blank\">NextScripts: Social Networks Auto-Poster "
-"(aka SNAP)</a>."
-msgstr ""
-
-#: includes/getting-started.php:43
-msgid "Other"
-msgstr ""
-
-#: includes/getting-started.php:45
-msgid "<strong>MicroPub</strong> - A MicroPub Server"
+"After you have established your identity, you can now log into the <a "
+"href=\"https://indieweb.org\">Indieweb wiki</a> and add a user page as a "
+"way of introducing yourself to the community. The wiki is also a great "
+"source of IndieWeb information"
 msgstr ""
 
-#: includes/getting-started.php:46
+#: includes/getting-started.php:25
 msgid ""
-"<strong>Indieauth</strong> - The plugin lets you login to the WordPress "
-"backend via IndieAuth. It uses the URL from the profile page to identify "
-"the blog user."
+"Install and activate the Webmentions and Semantic Linkbacks plugins. These "
+"will allow you to receive responses such as replies, likes, etc from other "
+"Indieweb sites"
 msgstr ""
 
-#: includes/getting-started.php:47
+#: includes/getting-started.php:26
 msgid ""
-"<strong>Indieweb Press This</strong> - An Indiewebified version of Press "
-"This"
+"You can now install the Bridgy plugin. It will add its settings options "
+"under Indieweb and allow you to directly register for Bridgy to connect "
+"your blog to responses from sites such as Facebook and Twitter too"
 msgstr ""
 
-#: includes/getting-started.php:48
+#: includes/getting-started.php:27
 msgid ""
-"<strong>Simple Location</strong> - Adds geographic/location support to "
-"WordPress"
+"Browse the Extensions page for a list of other recommended Indieweb plugins "
+"that can expand your functionality."
 msgstr ""
 
-#: includes/getting-started.php:53
+#: includes/getting-started.php:30
 msgid "Install Plugins"
 msgstr ""
 
-#: includes/getting-started.php:55
+#: includes/getting-started.php:32
 msgid "Themes"
 msgstr ""
 
-#: includes/getting-started.php:57
+#: includes/getting-started.php:34
 msgid ""
-"Some WordPress themes are compatible with microformats. The IndieWeb uses "
-"microformats2, the latest version, to mark up sites so that they can be "
-"interpreted by other sites when retrieved. Most parsers will fall back onto "
-"the older format if available."
+"The IndieWeb uses microformats2 to mark up sites so that they can be "
```

Comment:
Should this have a link to an IW wiki page for mf2? Is there a better introduction to mf2 somewhere else that doesn't get into plumbing?